src/common.styles.ts

```diff
@@ -112,6 +114,11 @@ export const useCommonStyles = makeStyles(theme => ({
         right: '40px',
         top: '40px',
         transform: 'translateY(-400px)',
+        [theme.breakpoints.down('sm')]: {
+            right: 20,
+            left: 10,
```

Comment:
Should the feedback box stay top right on a tablet? Maybe it's enough to remove `left` here. :thinking:

Reply:
@olav If I remove the left the feedback dialog will have no space from the left. It will look like this:
![image](https://user-images.githubusercontent.com/47088512/161011177-ad90f5ef-6c13-4716-8765-e780bf39cadc.png)

and this is how it looks like if I keep the left:
![image](https://user-images.githubusercontent.com/47088512/161011323-44f370c5-e710-4fb5-a469-6934e057e2e0.png)